### PR TITLE
Philip DeFraties

### PR DESCRIPTION
**Pull request by PhilipDeFraties**
Philip DeFraties

### CODE REVIEW

**Comment by PhilipDeFraties** [comment]
Philip DeFraties

File: lib/restaurant.rb [hunk 1]
```diff
@@ -1,37 +1,22 @@
...
   def initialize(opening_time, name)
     @opening_time = opening_time
     @name = name
     @dishes = []
   end
 
-  def add_dish(dish)
-    @dishes << dish
+  def closing_time(time)
+    (@opening_time.to_i + time).to_s + ':00'
```
Comment (timomitchel) [inline lib/restaurant.rb:11]: Nice job with the conversions and concatenation here.

File: lib/restaurant.rb [hunk 2]
```diff
@@ -1,37 +1,22 @@
...
   end
 
-  def closing_time(hours)
-    (@opening_time.to_i + hours).to_s + ':00'
+  def add_dish(dish)
+    @dishes << dish
   end
 
   def open_for_lunch?
     @opening_time.to_i <= 11
```
Comment (timomitchel) [inline lib/restaurant.rb:19]: In my opinion `@opening_time.to_i < 12` reads better based on the interaction pattern.

File: test/restaurant_test.rb [hunk 3]
```diff
@@ -51,4 +50,15 @@ def test_it_can_add_dishes
...
     assert_equal ['Burrata', 'Pizzetta', 'Ravioli'], restaurant.dishes
   end
+
+  def test_it_is_open_for_lunch
+
+    restaurant1 = Restaurant.new('10:00', 'Fuel Cafe')
+    restaurant2 = Restaurant.new('16:00', 'Il Poggio')
+
+    assert_equal true, restaurant1.open_for_lunch?
+    assert_equal  false, restaurant2.open_for_lunch?
```
Comment (timomitchel) [inline test/restaurant_test.rb:60]: Nice job with the setup and execution here.

**Pull request closed without merging**